### PR TITLE
Remove lwt multicore resolution

### PR DESCRIPTION
**Pull request by d4hines**
## Problem

We no longer need the custom lwt resoultion in package.json. We aren't using `Lwt_domain` for #147, and even if we did, `Lwt_domain` is on opam now.

## Solution

Remove it.

### CODE REVIEW

**Comment by d4hines** [comment]
Actually I think that when we do #250 we may want Lwt_domain after all so let's hold off on merging this

**Review by EduardoRFS — APPROVED** [review]
As discussed in private, it will not be needed in the immediate future.

anime girl dancing for a straightforward PR

**Pull request merged**
